model/error.yaml

```diff
@@ -0,0 +1,31 @@
+groups:
+  - id: error
+    type: attribute_group
+    prefix: error
+    brief: >
+      This document defines the shared attributes used to
+      report an error.
+    attributes:
+      - id: id
+        brief: 'Describes a class of error operation has ended with.'
+        type:
+          allow_custom_values: true
+          members:
+            - id: other
+              value: "_OTHER"
```

Comment:
I think prior discussions around `_OTHER` were limited to using it for cardinality capping.

I think that's essentially how it's used here also, so I'm good with this, just want to highlight it in case anyone has further thoughts.

Reply:
I use it for the same purpose here (e.g. instrumentation may support a few known exceptions/error codes and record everything else as `_OTHER`) and if we had metrics advice, we'd let users limit this set further.